src/emit.ts

```diff
@@ -62,6 +67,26 @@ export const recordEvent = (
     events[cid][event] = []
   }
 
+  // update vmodel
+  if (event.startsWith('update:')) {
+    const prop = event.slice(7)
+    const vmodel = vmodels?.get(prop)
+    if (vmodel) {
+      if (args.length !== 1) {
+        throw new Error(
+          'Two-way bound properties have to emit a single value. ' +
+            args.length +
+            ' values given.'
+        )
+      }
+      const v = args[0]
+      vmodel.value.value = v
+      vmodel.onChange?.(v)
+    } else {
+      // should we warn here?
```

Comment:
I would say no, because you could have props that have 2 way binding but not required.
In that case just I would just do:
```ts
if (!vmodel) {
    return;
}

if (args.length !== 1) {... etc
```
```